src/main/java/org/passay/dictionary/FileWordList.java

```diff
@@ -126,22 +120,16 @@ public FileWordList(
    * @throws  IllegalArgumentException  if cache percent is out of range.
    * @throws  IOException  if an error occurs reading the supplied file
    */
-  public FileWordList(
-    final RandomAccessFile raf,
-    final boolean caseSensitive,
-    final int cachePercent,
-    final CharsetDecoder decoder,
-    final boolean allocateDirect)
-    throws IOException
+  public FileWordList(final RandomAccessFile raf, final boolean caseSensitive, final int cachePercent,
+    final CharsetDecoder decoder, final boolean allocateDirect) throws IOException
```

Comment:
I actually prefer the long form, but I recognize I'm the only person in the world who thinks that way.

Reply:
Whatever you say, I realize it's a matter of style preference and I can remove this commit if you don't like it. Might be better to keep the codebase consistent with one style, though (and most of it is not one line per param). I'd personally prefer the established industry conventions... or I can just delete this commit, leave everything as it was and pretend it never happened :-)